clang/docs/checkedc/3C/CONTRIBUTING.md

```diff
@@ -1,7 +1,8 @@
 # Contributing to 3C
 
 Issues and pull requests related to 3C should be submitted to [CCI's
-`checkedc-clang` repository](https://github.com/correctcomputation/checkedc-clang), not
+`checkedc-clang`
+repository](https://github.com/correctcomputation/checkedc-clang), not
```

Comment:
Also non optimal.

Reply:
Yes, I don't know of a good solution to this problem in Markdown files.  Let's hope most readers are using rich viewers; we do use some Markdown features that are specific to GitHub's renderer.